source/administration/changelog.md

```diff
@@ -4,6 +4,97 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
 
 Also see [changelog in progress](https://bit.ly/2nK3cVf) for the next release.
 
+## Release v5.34 - [Feature Release](https://docs.mattermost.com/administration/release-definitions.html#feature-release)
+
+**Release Day: 2021-04-16**
+
+### Combatibility
+
+### Breaking Changes
+ 
+**IMPORTANT:** If you upgrade from a release earlier than v5.33, please read the other [Important Upgrade Notes](https://docs.mattermost.com/administration/important-upgrade-notes.html).
+
+### Highlights
+
+#### Added pre-packaged Incident Collaboration 1.5.0 (E20)
+ - Users can now configure playbooks to perform specific actions automatically when an incident starts, such as inviting members to the incident channel and assigning a commander.
+
+### Improvements
+
+#### User Interface (UI)
+ - Removed Beta tags from Swedish and Bulgarian languages.
+ - System Admins are now prompted when joining a private channel via a permalink.
+ - Added support for adding in-product notices for external dependency deprecation details.
+ - Improved the timezone selector component.
+ - Introduced a new theme variable for the team sidebar.
+ - Added support for automatic right-to-left (RTL) detection in browsers.
+ - Updated the font size for the **Add People** channel modal.
+ - Online status is now shown in the channel switcher.
+ - Improved the design and layout of email notifications for password resets, member invites, and member welcome and verifications.
+
+#### Administration
+ - Added mmctl commands to create, list, download and delete export files.
+ - Added schema migrations phase 0 (``Teams``, ``TeamMembers``).
+ - Removed references to ``SqlLite3`` from the code.
+ - Bleve updates are now logged in the config only when there is an actual change in the ``BleveSettings`` instead of on every config update.
+ - Profiling the Mattermost server with pprof is now available for Team Edition.
+ - Added attributes to split.io feature flags.
+
+### Bug Fixes
+ - Fixed unsafe access of properties of the plugin environment during ``ServePluginPublicRequest``.
+ - Fixed an issue where the Admin Console > Server Logs did not focus to the sidebar filter upon reload.
+ - Fixed an issue where the Gif picker appeared empty instead of showing a “No results” modal when no results were displayed.
+ - Fixed an issue where the keyboard accessibility controller was not allowed to resume left-hand side scroll after drag and drop.
+ - Fixed an issue where markdown links rendered incorrectly.
+ - Fixed an issue where the slack theme import failed due to changes in formatting of Slack export color schemes.
+ - Fixed an issue where tooltips were missing for channels with a long name.
+ - Fixed a race condition which would crash the app server due to improper handling of websocket closing.
```

Comment:
should "websocket" be pluralized? "due to improper handling of websockets closing"?

Reply:
```suggestion
 - Fixed a race condition which would crash the app server due to improper handling of WebSocket closing.
```